File: indic-script-converter-dart/tools/latn_iast_transliteration_verification/gujr_to_deva_test.dart
// ignore_for_file: avoid_print

import 'dart:convert';

import 'package:indic_script_converter/deva_gujr_converter.dart';
import 'example_gujr.dart' as examples;

void main() {
  print('----------------------------------------------------------------');
  print(' GUJARATI TO DEVANAGARI SCRIPT CONVERSION');
  print('----------------------------------------------------------------');

  for (final source in examples.gujaratiSmokeSamples) {
    final result = source.toCanonicalDevanagariFromGujarati();
    print('"$source" -> "$result"');
  }
}

File: indic-script-converter-python/tools/latn_iast_transliteration_verification/gujr_to_deva_test.py
from __future__ import annotations

from indic_script_converter.deva_gujr_converter import (
    to_canonical_devanagari_from_gujarati,
)
from indic_script_converter.tools.example_gujr import gujarati_smoke_samples


def main() -> None:
    print('----------------------------------------------------------------')
    print(' GUJARATI TO DEVANAGARI SCRIPT CONVERSION')
    print('----------------------------------------------------------------')

    for source in gujarati_smoke_samples:
        result = to_canonical_devanagari_from_gujarati(source)
        print(f'"{source}" -> "{result}"')


if __name__ == '__main__':
    main()
